### ISSUE TITLE
App crashes when turning on last.fm with self-signed certificates

### ISSUE THREAD

**Issue by sigma67** [adlerluiz/ytmdesktop#1408]
### Consent

- [X] I verified that there is no open/closed issue for the same subject.
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Current Behavior

```
YouTube Music Desktop App Crashed

Environment Details:
    YouTube Music Desktop App 2.0.5
    Electron 31.2.1
    Windows NT 10.0.22621

Name: TypeError
Message: fetch failed
Cause: Error: self signed certificate in certificate chain

TypeError: fetch failed
    at node:internal/deps/undici/undici:12502:13
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async t.default.getSession (C:\Users\...\AppData\Local\youtube_music_desktop_app\app-2.0.5\resources\app.asar\.webpack\main\index.js:8:453032)
```

### Expected Behavior

No crash

### Steps To Reproduce

1. Have a self-signed cert in your chain (i.e. corporate environment)
2. Turn on last.fm integration
3. App will crash instantly with above error
4. App will also crash on every startup from that point on

### YTMDesktop

latest main

### OS

Windows

### OS Version

11

### Arch

x64

### Installation way

first from latest release, then from latest main artifact

**Comment by NovusTheory** [comment]
Closing this as a slight duplicate of #1275

tl;dr from discussions

I'm not interested in providing a "continue anyway" sort of system as this also isn't part of the main browser area of the app and is much more nuanced to program with no security benefit of doing so. If on a corporate network you should seek to have a proper proxy CA and that clients trust that CA instead of self signing certificates.

The crashing is a side effect that just won't be fixed immediately because of how this errors comes up and isn't a priority
